### PR TITLE
Enhance some CLI Command Descriptions

### PR DESCRIPTION
**Pull request by lborgav**
I’ve updated the command descriptions that were previously using Cobra’s default placeholder text.
The new descriptions provide more clarity and correctness

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -33,13 +33,9 @@ import (
...
 	Use:   "deploy",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Deploy a new version of your application to your VPS using Sidekick",
+	Long: `This command deploys a new version o your application to your VPS.
```
Comment (MightyMoud) [inline cmd/deploy.go:37]: Small typo here `of your application`

**Review by MightyMoud — CHANGES_REQUESTED** [review]
Absolute Legend! This been on my task list for a while now. 
Tiny comment and LGTM.

Thanks @lborgav

File: cmd/deploy.go (hunk 1 above)
Comment (lborgav) [reply cmd/deploy.go]: Just fixed it. 
Nice and helpful project. I'd happy to help

Comment (MightyMoud) [reply cmd/deploy.go]: Thank you! 
Happy to hear that! There will be big features we would like to build in the next couple of months. We can discuss more

**Pull request merged**